### PR TITLE
created first draft of templates

### LINKED ISSUE #16: Add corresponding templates that mirror the handbook processes
### Feature
TODO: Add the first draft of templates as discussed in the Gold Standard meetings and handbook discussions
### Files
new file:   .github/ISSUE_TEMPLATE/bug_report.md
	new file:   .github/ISSUE_TEMPLATE/cert_feedback.md
	new file:   .github/ISSUE_TEMPLATE/certification.md
	new file:   .github/ISSUE_TEMPLATE/decision.md
	new file:   .github/ISSUE_TEMPLATE/feature_req.md
	new file:   .github/ISSUE_TEMPLATE/handbook_mod.md
	new file:   .github/ISSUE_TEMPLATE/issue.md
	new file:   .github/ISSUE_TEMPLATE/issue_review.md
	new file:   .github/ISSUE_TEMPLATE/milestone.md
	new file:   .github/ISSUE_TEMPLATE/milestone_review.md
	new file:   .github/ISSUE_TEMPLATE/peer_review.md
	new file:   .github/ISSUE_TEMPLATE/proj_review.md
	new file:   .github/ISSUE_TEMPLATE/project.md
	new file:   .github/ISSUE_TEMPLATE/quantative_review.md
	new file:   .github/ISSUE_TEMPLATE/rank_vote_prop.md
	new file:   .github/ISSUE_TEMPLATE/rep_immed_act.md
	new file:   .github/ISSUE_TEMPLATE/task.md
	new file:   .github/ISSUE_TEMPLATE/template_mod.md
	new file:   .github/PULL_REQUEST_TEMPLATE/changelog.md
	new file:   .github/PULL_REQUEST_TEMPLATE/decision.md
	new file:   .github/PULL_REQUEST_TEMPLATE/handbook_mod.md
	new file:   .github/PULL_REQUEST_TEMPLATE/issue.md
	new file:   .github/PULL_REQUEST_TEMPLATE/milestone.md
	new file:   .github/PULL_REQUEST_TEMPLATE/project.md
	new file:   .github/PULL_REQUEST_TEMPLATE/rank_vote_prop.md
	new file:   .github/PULL_REQUEST_TEMPLATE/rep_immed_act.md
	new file:   .github/PULL_REQUEST_TEMPLATE/task.md
	new file:   .github/PULL_REQUEST_TEMPLATE/template_mod.md
	new file:   .github/QA_TEMPLATE/cert_feedback.md
	new file:   .github/QA_TEMPLATE/certification.md
	new file:   .github/QA_TEMPLATE/qa.md
	new file:   .github/QA_TEMPLATE/qa_strategy.md
	new file:   .github/REVIEW_TEMPLATE/issue_review.md
	new file:   .github/REVIEW_TEMPLATE/milestone_review.md
	new file:   .github/REVIEW_TEMPLATE/peer_review.md
	new file:   .github/REVIEW_TEMPLATE/proj_review.md
	new file:   .github/REVIEW_TEMPLATE/quantative_review.md

### PR DESCRIPTION
**Pull request by andrewnguyen22**
closes #16

### CODE REVIEW

**Review by oten91 — APPROVED** [review]
Looks good! . I think we should actually re-review them when the organization handbook version is out to keep them in line.

**Comment by emedrano2112** [comment]
LGTM. We could sync up a meeting to go over the details real quick and open the door for little improvements (if they exist).

**Review by aqt01 — APPROVED** [review]

**Review by edjroz — APPROVED** [review]

File: .github/QA_TEMPLATE/certification.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+**Certification Contract**
+
+- [ ] The certification of this deliverable ensures Stability Level as described in the handbook
+
+**Justification Statement**
+
+[Provide a detailed statement of satisfaction of the Certification Level requirements]
+
+### Alternatives Considered
```
Comment (emedrano2112) [inline .github/QA_TEMPLATE/certification.md:34]: I would add a result's section. The idea would be to have the test results as evidence, and not leave it to just words. Good examples would be, providing test execution data (if applicable), test cases statuspassed/failed/skipped/blocked, test scenario status (passed/failed/skipped/blocked), etc.

File: .github/QA_TEMPLATE/qa.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+**Conclusion**
+
+[Conclusion drawn from investigative research]
+
+**External Links**
+
+[If Applicable]
+
+### General Notes
```
Comment (emedrano2112) [inline .github/QA_TEMPLATE/qa.md:169]: I would add a result's section. The idea would be to have the test results as evidence, and not leave it to just words. Good examples would be, providing test execution data (if applicable), test cases statuspassed/failed/skipped/blocked, test scenario status (passed/failed/skipped/blocked), etc.

**Review by luyzdeleon — APPROVED** [review]
Approving, the explanations on where all these templates will be used will live in the handbook.

File: .github/ISSUE_TEMPLATE/cert_feedback.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+name: Certification Feedback
+about: The Certification Feedback Issue Template is high level formal documentation of a feedback made by the reviewer during the certification process. 
+title: "Certification Feedback: [TITLE OF THE MILESTONE]:[REVIEWER]"
+labels: CERT FEEDBACK
+assignees: ''
```
[the remark this answers is not in the document]
Comment (andrewnguyen22) [reply .github/ISSUE_TEMPLATE/cert_feedback.md:6]: It's an issue default

File: .github/ISSUE_TEMPLATE/handbook_mod.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+[TITLE OF THE HANDBOOK MODIFICATION PROPOSAL]
+### Date(s)
+01/01/1970
+### Section
+- [ ] Decision Making Framework
+- [ ] Self Improving Organization
+- [ ] Accountability
+- [ ] Development Lifecycle
+- [ ] Development Pace
+- [ ] Emergency
```
[the remark this answers is not in the document]
Comment (andrewnguyen22) [reply .github/ISSUE_TEMPLATE/handbook_mod.md:19]: I'd rather have people fill it in themselves besides checking 'other'

File: .github/ISSUE_TEMPLATE/milestone.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+name: Milestone
+about: The Milestone Template is high level formal documentation of a Milestone created in the context of Pocket Network Corporation's Development Lifecycle. 
+The purpose of this document is to be written proof of a Milestone by the Pocket Network Corporation members for the PNI audit trail.  
+title: "Milestone: [TITLE OF THE MILESTONE]"
+labels: MILESTONE
+assignees: ''
+---
+### Milestone Title [TITLE OF THE MILESTONE]
```
[the remark this answers is not in the document]
Comment (andrewnguyen22) [reply .github/ISSUE_TEMPLATE/milestone.md:9]: I disagree, this is going to be an issue title

It's weird to see an issue that's

Title: RC-0.6.0

Rather 

Milestone Title: RC-0.6.0

**Review by julio-jimenez — APPROVED** [review]

**Review by wgarcia4190 — APPROVED** [review]

**Review by emedrano2112 — APPROVED** [review]
My comments were resolved. I approve this changes.

**Review by nelcolon — APPROVED** [review]

**Pull request merged**